profiles/killbill/src/test/resources/killbill.properties

```diff
@@ -21,6 +21,8 @@ org.killbill.overdue.uri=overdue.xml
 
 org.killbill.payment.retry.days=8,8,8
 
+org.killbill.payment.plugin.timeout=8s
```

Comment:
The default timeout of the java client is 10 seconds, so I've lowered the threshold in kb (from the default 30s), so we could add a test case for timeouts.